cypress.json

```diff
@@ -14,7 +14,8 @@
     "SECURITY_ENABLED": false,
     "username": "admin",
     "password": "admin",
-    "ENDPOINT_WITH_PROXY": false
+    "ENDPOINT_WITH_PROXY": false,
```

Comment:
what is the difference between ENDPOINT_WITH_PROXY and MANAGED_SERVICE_ENDPOINT ?

Reply:
I think for the `MANAGED_SERVICE_ENDPOINT` flag we are trying to separate the tests that are only applicable to managed service side domain. In this case the Amazon SNS type destination is only on AWS Opensearch so it will fail on an open source domain. I believe there are cases when the domain is on managed service side and is not proxy enabled, so it could be 2 different flags.  

I will add the description of this flag to the documentation too.